src/main/java/net/minecraftforge/event/world/FoundChunksForSpawningEvent.java

```diff
@@ -0,0 +1,69 @@
+package net.minecraftforge.event.world;
+
+import javax.annotation.Nullable;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import net.minecraft.entity.EnumCreatureType;
+import net.minecraft.util.math.ChunkPos;
+import net.minecraft.world.WorldServer;
+import net.minecraftforge.fml.common.eventhandler.Event;
+
+public class FoundChunksForSpawningEvent extends Event
+{
+    private final FoundChunksForSpawningEventBuilder builder;
+    private ImmutableList<ChunkPos> eligibleChunksForSpawning;
+    private ImmutableMap<EnumCreatureType, FoundChunksForSpawningEventCreatureTypeData> creatureTypeData;
+
+    public FoundChunksForSpawningEvent(FoundChunksForSpawningEventBuilder builderIn)
+    {
+        builder = builderIn;
+    }
+
+    public WorldServer getWorldServer()
+    {
+        return builder.worldServer;
+    }
+
+    /**
+     * Usually the number of eligible chunks for spawning, but sometimes greater, depending on something about the world border and the player chunk map. A value of -1 indicates
+     * that there was no attempt to calculate this number.
+     */
+    public int getI()
+    {
+        return builder.i;
```

Comment:
This is the name of the variable in the vanilla code. I added comments to describing what I could glean from the code, but I'm hesitant to rename it because I'm not sure exactly what the author(s) indented for it to mean.

But if you like, I can take a guess... maybe something like "numAttemptedEligibleChunksForSpawning"?

Reply:
Fixed at b6c389a937e2fe30ea1f55457c1d5806bedcd0e8